cmd/runtimetest/main.go

```diff
@@ -320,6 +322,95 @@ func validateROPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func getIDMappings(idType string) ([]rspec.IDMapping, error) {
+	var mapFile string
+	var idMaps []rspec.IDMapping
+	switch idType {
+	case "uid_map":
+		mapFile = "/proc/1/uid_map"
+	case "gid_map":
+		mapFile = "/proc/1/gid_map"
```

Comment:
And I'd bump this choice up to the caller and have `getIDMappings` take a path:

    func getIDMappings(path string) ([]rspec.IDMapping, error)